describe("Planned error tests", function () {
  it("Check the work to throw an error", function () {
  //  throw new Error(`Planned error`);
  });
});
